src/flake8/main/options.py

```diff
@@ -272,6 +273,16 @@ def register_default_options(option_manager):
         " For example, ``--select=E4,E51,W234``. (Default: %(default)s)",
     )
 
+    add_option(
+        "--extend-select",
+        metavar="errors",
+        default="",
+        parse_from_config=True,
+        comma_separated_list=True,
+        help="Comma-separated list of errors and warnings to add to the list"
+        " of selected ones. For example, ``--extend-select=E4,E51,W234``.",
```

Comment:
I would write this like:
```python
help=(
    "part one of the string "
    "part 2 of the string."
),
```

just so the words line up

Reply:
Since this is the style used for all other help messages, I would go with consistency here and not change it.

Reply:
please fix this, it's one fewer for me to fix later